<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardWidget</class>
 <widget class="QWidget" name="KeyboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="keyboardImage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>236</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../data/data_res.qrc">:/c64_images/C64C_cropped.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keySpace">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyLeft">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="key0">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyPlus">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyMinus">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyPound">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyHome">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyDel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyF1">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyF3">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyF5">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>100</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyF7">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>140</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyCtrl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyQ">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyW">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyE">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyR">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyT">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyY">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyU">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyI">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyO">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyP">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyAt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyAsterisk">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyUp">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>50</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyRestore">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyRunStop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyA">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyS">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyD">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyF">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyG">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyH">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyJ">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyK">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyL">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyColon">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keySemicolon">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyEquals">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyReturn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyCommodore">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyLShift">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyZ">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyX">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyC">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyV">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyB">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyN">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyM">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyComma">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyPeriod">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keySlash">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyRShift">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyCursorDown">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyCursorRight">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>140</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="resetKeysButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>reset Keys</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="keymapCheckBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>keymap</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../../data/data_res.qrc"/>
 </resources>
 <connections/>
</ui>
